From a0ea7d7efc31a38ef73d70a42a3244524c548070 Mon Sep 17 00:00:00 2001
From: Kestrel <ke5trel@protonmail.com>
Date: Thu, 23 May 2019 10:44:29 +0000
Subject: [PATCH] Bug 1544228 - Fix some about: pages text color contrast for
 GTK dark system themes r=dao

Some about: pages use -moz-Dialog background color which is dark with popular GTK dark themes (Adwaita-dark, Yaru-dark) so the text color needs to be set to -moz-DialogText to retain contrast. This affects about:buildconfig, about:cache, about:serviceworkers and about:credits.

Differential Revision: https://phabricator.services.mozilla.com/D31414
---
 toolkit/themes/shared/about.css | 1 +
 1 file changed, 1 insertion(+)

diff --git a/toolkit/themes/shared/about.css b/toolkit/themes/shared/about.css
index 12fb65fe7271..aa89f01e5aa0 100644
--- a/toolkit/themes/shared/about.css
+++ b/toolkit/themes/shared/about.css
@@ -4,6 +4,7 @@
 
 html {
   background: -moz-Dialog;
+  color: -moz-DialogText;
   padding: 0 1em;
   font: message-box;
   line-height: 1.6em;
